src/content/docs/infrastructure/host-integrations/host-integrations-list/nuxt-js-integration.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Nuxt.js integration
+tags:
+  - nuxt.js integration
+  - nuxt integration
+  - new relic integrations
+translate:
+  - jp
+  - kr
+metaDescription: "Learn how to query your New Relic data with NRQL, our SQL-like query language."
+redirects:
+---
+With our Nuxt.js dashboard, you can easily track your application error rate, initial page load average, route changes, throughput, and page views with JavaScript errors. Built with our infrastructure agent and our Vue browser monitoring, the Nuxt.js dashboard lets you view critical information all in one place.
+
+## Install infrastructure agent [#infra]
+Before you can get your data into New Relic, first install the infrastructure agent (if you have it installed, you can skip this step). This enables New Relic to collect data so that application error rates, throughput, and page views with JavaScript errors can appear in dashboards.
+
+There are two ways to install the infrastructure agent:
+* Follow the [guided install](https://one.newrelic.com/marketplace?state=607786be-99a9-fa66-4b67-3f7f3b9551d4) to instrument our system with the infrastructure agent.
+* [Install the infrastructure agent](docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) using the command line. 
+
+
+## Install Vue browser monitoring [#install]
+After installing the infrastructure agent, the next step is to install the browser monitor. From **[one.newrelic.com](https://one.newrelic.com)**, click **Add data > Browser & mobile > Vue**.
+Follow the instructions from that screen to generate the JavaScript code snippet to use during configuration.
```

Comment:
Do we want to have a second screenshot that will be similar to the first one we put in? I'm totally fine with it, it just feels like it breaks consistency with some of the other docs we have that I've seen (though it could be consistent with others I haven't seen!).

Reply:
Hmm. Maybe I can put it in as a second iteration just to see if it looks ok, but for this PR let's skip it for now. :) I got a demo that similarly uses the browser agent and I'm so/so on whether I want to add screenshots for it yet.